src/callbox/renderer/CallboxApp.vue

```diff
@@ -114,6 +114,23 @@ function dismiss() {
 	</AppWindow>
 </template>
 
+<style>
+:root {
+	--join-call-background-color: var(--color-border-success);
```

Comment:
should be used `var(--color-element-success);` instead of border, also in spreed repo

Reply:
I'll fix when it is fixed in `spreed` to be consistent. Or when it is fixed globally.